ReactNativeClient/lib/fs-driver-node.js

```diff
@@ -160,6 +160,15 @@ class FsDriverNode extends FsDriverBase {
 		}
 	}
 
+	async readDir(path) {
+		try {
+			// return fs.readdirSync(path, {withFileTypes: true}).map(e => e.name); // From Node v10
+			return fs.readdirSync(path); // For Node v8
+		} catch (error) {
+			throw this.fsErrorToJsError_(error, path);
+		}
+	}
+
```

Comment:
To get the files inside a directory, please use `readDirStats()`.

Reply:
Ah yes, I didn't notice this function. Maybe the naming of it could just be `readDir()`, or `readDirFiles()`. It's not immediately clear what "stats" mean in the context of a directory.